src/CMake/config/postinst-aws.in

```diff
@@ -1,36 +1,39 @@
 #!/bin/sh
 
-if [ -n "`dkms status -m xrt-aws -v @XRT_VERSION_STRING@`" ]; then
-    echo "Unloading old XRT Linux kernel modules"
-    modprobe -r awsmgmt
+nm /opt/xilinx/xrt/lib/libaws_mpd_plugin.so |grep 'fpga_mgmt' > /dev/null 2>&1
```

Comment:
Should we use a more deterministic way to determine if a plugin links fpga mgmt library like using ldd?

Reply:
the aws mpd plugin, as well as awssak, links the fpga mgmt library as .a (static linking) not .so, ldd is not working here.
I don't find a more decent way to determine whether the pkg is being installed on Pegasus or F1 than the current one(checking symbols or .so files)

Reply:
@sonals 
Do u still have other comments?

So far, the static linking way is easier for us to build and deliver xrt pkg, otherwise, we also need to think about how to install in the xrt the fpga mgmt .so, which itself isn't delivered as a pkg.